public/js/app/helpers/Handlebars.js

```diff
@@ -18,7 +18,18 @@ define(["app/app", "ember"], function(App, Ember) {
     //   collapseEffect: 'fadeOut'
     // })
 
-    return new Ember.Handlebars.SafeString(text.html())
+    var html = text.html()
+
+    // make links from @login
+    var parts = html.split(/(<.*?>)/g), i
+    for (i = 0; i < parts.length; i += 4) {
+      parts[i] = parts[i].replace(/\B@([a-z0-9]+)/g, function(u) {
+        return '<a href="/' + u.substr(1) + '">' + u + '</a>'
```

Comment:
This is going to reload an entire ember application, e.g. transitionToRoute will be better (not sure it's safe to use it in Handlebars helper though).